code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -371,6 +371,7 @@
 	return TRUE
 
 /datum/dynamic_ruleset/roundstart/revs/execute()
+	status = RULESET_FAILED
```

Comment:
Won't this make this get cleaned up if any sleeps happen ? Should be waiting or some setting up state ?

Reply:
I rewrote the roundstart execute to be like the mid/latejoin one. Every rule is now ran on a timer to a proc in gamemode which executes instead of executing on a timer. Gives a bit more control and allows us to work off return values for delayed rulesets as well, so status is gone. Its nicer, more consistent and should be more robust.
Since we no longer have to pass along info from rulesets to gamemode, we don't add the ruleset to the processing list before execute either.